### PR TITLE
ref: correct dvctable highlighter

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

File: content/docs/command-reference/exp/remove.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ $ dvc exp remove e41d5b4 split16
 $ dvc exp show --include-params=train.min_split --no-pager
 ```
 
-```table
+```dvctable
 ┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
 ┃ Experiment            ┃ Created      ┃ State  ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
```
Comment (shcheklein) [inline content/docs/command-reference/exp/remove.md:90]: should we highlight the header (colorize params, metrics, etc)?

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/exp/remove.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/exp/remove.md:90]: Idk how to do that ⏳

Comment (jorgeorpinel) [reply content/docs/command-reference/exp/remove.md]: OK done in d866b025 (I think)

File: content/docs/command-reference/exp/remove.md [hunk 2]
```diff
@@ -85,9 +85,9 @@ $ dvc exp remove e41d5b4 split16
 $ dvc exp show --include-params=train.min_split --no-pager
 ```
 
-```table
+```dvctable
 ┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment            ┃ Created      ┃ State  ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
+┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃ neutral:**State**  ┃ metric:**avg_prec** ┃ metric:**roc_auc** ┃ metric:**train.min_split** ┃
```
Comment (jorgeorpinel) [inline content/docs/command-reference/exp/remove.md:90]: > should we highlight the header (colorize params, metrics, etc)?

Sanity checked in https://dvc-org-ref-dvctable-ojpnqwjsg.herokuapp.com/doc/command-reference/exp/remove#examples:

image

Comment (jorgeorpinel) [reply content/docs/command-reference/exp/remove.md:90]: The scroll bar isn't ideal but it's a problem in most `exp show` tables now (it looked better when they were images but wasn't copy-pasteable).

File: content/docs/user-guide/experiment-management/cleaning-experiments.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ $ dvc exp show --all-branches
 
 ```dvctable
 ┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
-┃ white:**Experiment**    ┃ white:**Created**      ┃    yellow:**acc** ┃ blue:**model.conv_units** ┃
+┃ neutral:**Experiment**              ┃ neutral:**Created**      ┃    metric:**acc** ┃ param:**model.conv_units** ┃
 ┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/cleaning-experiments.md:82]: Checked in https://dvc-org-ref-dvctable-ojpnqwjsg.herokuapp.com/doc/user-guide/experiment-management/cleaning-experiments.

image

**Pull request merged**